### PR TITLE
DTSAM-575 `CIVIL_WA_2_0` configure `Deputy District Judge` appointment

### PR DESCRIPTION
**Pull request by abul-syed**
### Jira link

https://tools.hmcts.net/jira/browse/DTSAM-575

### Change description

Civil expand JOH rules to cover `Deputy District Judge` and appointment type `Fee-paid`

### CODE REVIEW

**Review by mikebrownccd — APPROVED** [review]
LGTM

**Review by mattnayler — DISMISSED** [review]

File: src/test/java/uk/gov/hmcts/reform/orgrolemapping/domain/service/DroolCivilHearingJudicialRoleMappingTest.java [hunk 1]
```diff
@@ -94,6 +94,11 @@ static Stream<Arguments> endToEndData() {
                         true,
                         List.of(""),
                         List.of("judge", "fee-paid-judge", "hmcts-judiciary", "hearing-viewer")),
+                Arguments.of("Deputy District Judge",
```
[the remark this answers is not in the document]
Comment (abul-syed) [reply src/test/java/uk/gov/hmcts/reform/orgrolemapping/domain/service/DroolCivilHearingJudicialRoleMappingTest.java:97]: addressed point 1 see [90f5a00](https://github.com/hmcts/am-org-role-mapping-service/pull/2128/commits/90f5a009e9d581fb5f028616bbd3112882ec4d05)

**Review by rajkatla-hmcts — APPROVED** [review]

**Review by mattnayler — APPROVED** [review]
Re-approved after test change

**Review by mikebrownccd — APPROVED** [review]
LGTM

**Pull request merged**